lxd/api_cluster.go

```diff
@@ -571,7 +571,7 @@ func clusterPutJoin(d *Daemon, r *http.Request, req api.ClusterPut) response.Res
 		var projects []dbCluster.Project
 
 		err = d.db.Cluster.Transaction(context.TODO(), func(ctx context.Context, tx *db.ClusterTx) error {
```

Comment:
The commit message " lxd/warnings/warnings: Remove usage of empty filter structs " for this commit is invalid as it covers a lot more than that.

Reply:
Fixed.